code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -107,31 +93,94 @@
 	wings = bee_icons["[icon_base]_wings"]
 	overlays += wings
 
-
 //We don't attack beekeepers/people dressed as bees//Todo: bee costume
 /mob/living/simple_animal/hostile/poison/bees/CanAttack(atom/the_target)
 	. = ..()
+	return .
+
+/mob/living/simple_animal/hostile/poison/bees/Found(atom/A)
+	if(ishuman(A))
```

Comment:
this top if (and the contents of it) are unneeded, humans are ```/living``` and their version of the proc will be automatically called.